src/code/fault.c

```diff
@@ -1206,7 +1202,6 @@ void Fault_ThreadEntry(void* arg) {
         } else {
             // Draw error bar signifying the crash screen is available
             Fault_DrawCornerRec(GPACK_RGBA5551(255, 0, 0, 1));
-            Fault_WaitForButtonCombo();
```

Comment:
If you are going to remove the call here entirely, then you should also remove the function itself.